src/registrar/migrations/0123_alter_domaininformation_submitter_and_more.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 4.2.10 on 2024-08-29 23:17
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("registrar", "0122_create_groups_v16"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="domaininformation",
+            name="submitter",
+            field=models.ForeignKey(
+                blank=True,
+                help_text='Person listed under "your contact information" in the request form',
+                null=True,
+                on_delete=django.db.models.deletion.SET_NULL,
```

Comment:
Set submitter to null on delete since we'll be deleting submitter as a field altogether in 0125